frontend/src/components/ExportChat/ExportContent.tsx

```diff
@@ -8,21 +8,22 @@ import {
 } from '@react-pdf/renderer';
 
 import CombinedFonts from '@src/assets/fonts/CombinedFonts.ttf';
-import NotoSansSC from '@src/assets/fonts/NotoSansSC-Regular.ttf';
-import NotoSerifJP from '@src/assets/fonts/NotoSerifJP-Regular.otf';
 import { ChatMessage } from '@src/models/chat';
 import { EmailInfo } from '@src/models/email';
 import { LEVEL_NAMES } from '@src/models/level';
 
 import ExportChatBox from './ExportChatBox';
 import ExportEmailBox from './ExportEmailBox';
 
+/*
+Chinese and Japanese font support: currently we're using a Combined font pack,
+but these two font packs are huge so they're kept separate. Only register one
+font pack at a time, then set fontFamily to match in the stylesheet beneath.
+*/
+//Font.register({ family: 'NotoSerifJP', src: NotoSerifJP });
+//Font.register({ family: 'NotoSansSC', src: NotoSansSC });
```

Comment:
Actually, if we're going to leave this example code in, then maybe worth doubling down and mentioning explicitly in the comment above that the necessary files must be downloaded.

Reply:
This is for running locally, so unless we remove the fonts from the repo, they don't need to be downloaded.

Reply:
Fonts now removed, so README and comment updated.